TypeCobol.Test/Parser/Preprocessor/ReplaceTestFiles/PgmEmptyPartialWordReplace2.cpy

```diff
@@ -0,0 +1,13 @@
+﻿001390*REPLACE ==:-MAJ:== BY ==-TRACE==.                                01340000
```

Comment:
Can you also create a test with a full program under "TypeCobol.Test/Parser/Programs/Cobol85/" just to be sure this PR integrates well with the rest ?